src/espresso-migration/UpdateChainConfig.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.8.0;
+// Example action contract to upgrade chain config to turn on espresso mode
+
+import "../precompiles/ArbOwner.sol";
+
+contract SetEspressoChainConfig {
+    function perform(string calldata serialiazedEspressoConfig) public {
```

Comment:
can anyone call this function or should we add some access control here?

Reply:
The setChainConfig function has built in access control, but I think it could be beneficial to add some additional access control to this function as well.

I think a good approach would be to add a custom error and a revert based on the ArbOwner.isChainOwner function.